dataverse-persistence/src/main/resources/db/migration/V29__changeUserNotificationType.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE usernotification ALTER COLUMN type TYPE varchar(255);
```

Comment:
Watch out for migration numberring. V29 is taken

Reply:
Ohh, by whom? There is nothing on develop.

Reply:
V29__addSourceFieldToDatasetField.sql